ontologies/vocabs.ttl

```diff
@@ -2473,6 +2487,62 @@ There is not enough information available to determine the type of threat actor.
 	skos:prefLabel "unknown"^^xsd:string ;
 	.
 
+gist:_WindowsServiceStatus_service-continue-pending
+	a gist:WindowsServiceStatus ;
+	skos:definition """STIX 2.1 description: 
+The service continue is pending."""^^xsd:string ;
+	skos:prefLabel "SERVICE_CONTINUE_PENDING"^^xsd:string ;
+	gist:stixTerm "SERVICE_CONTINUE_PENDING"^^xsd:string ;
+	.
```

Comment:
IRI label mismatch.
'service' in IRI & label is redundant. Change all instances.
```suggestion
gist:_WindowsServiceStatus_continue-pending
	a gist:WindowsServiceStatus ;
	skos:definition """STIX 2.1 description: 
The service continue is pending."""^^xsd:string ;
	skos:prefLabel "continue pending"^^xsd:string ;
	gist:stixTerm "SERVICE_CONTINUE_PENDING"^^xsd:string ;
	.